test/functional/Java19andUp/build.xml

```diff
@@ -36,6 +36,17 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml" />
 	<property name="TestUtilities" location="../TestUtilities/src"/>
 
+	<if>
+		<matches string="${JDK_VERSION}" pattern="^(19|20)$$"/>
+		<then>
+			<!-- Loom is a preview feature in Java 19 and 20. -->
+			<property name="preview" value="--enable-preview --source ${JDK_VERSION}" />
+		</then>
+		<else>
+			<property name="preview" value="" />
+		</else>
+	</if>
+
```

Comment:
On further reflection, I don't see a problem using `--enable-preview` unconditionally. Even after Loom ceases to be a preview feature, compiling with `--enable-preview` should continue to be successful.

Also, besides being more compact, I suggest we should avoid using ant-contrib features unnecessarily, so if we **_needed_** to have a conditional property value, it should be written in this form:
```
		<condition property="preview" value="--enable-preview" else="">
			<matches string="${JDK_VERSION}" pattern="^(19|20)$$" />
		</condition>
```